api/api-doc/storage_service.json

```diff
@@ -2726,6 +2726,22 @@
             }
          ]
       },
+      {
+         "path":"/storage_service/quiesce_topology",
```

Comment:
What is the definition of the topology to be quiesced? The code wait for an ongoing operation to complete, but it does not wait for request queue to be empty and if it is not the next op can start immediately. OTOH even if the queue is empty a new request can be summited immediately after we check.

Also the info is available through CQL, though unlike here, it has to be polled instead of waited, but it may be good enough for tests.

Reply:
> What is the definition of the topology to be quiesced? 

Topology operations which were ongoing before the call are finished after the call. This doesn't consider requested but not started operations. I don't need it for my use case, but we may consider including them to make the wait stronger.

> Also the info is available through CQL, though unlike here, it has to be polled instead of waited, but it may be good enough for tests.

Having it as a call is more efficient than polling because it can wait on topology event, and we don't have to deal with the tradeoff between low latency (better with small poll period) and overhead (worse with small poll period).

It may be useful outside tests too, for admin operations/scripts.